sdk/python/foundation-models/system/finetune/Llama-notebooks/distillation/distillation_qanda.ipynb

```diff
@@ -0,0 +1,609 @@
+    "ai_project = ml_client._workspaces.get(ml_client.workspace_name)\n",
+    "ai_project._workspace_id"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Pick a teacher model\n",
+    "\n",
+    "We support **Meta-Llama-3.1-405B-Instruct** as the teacher model. \n",
+    "### First deploy the teacher model in Azure AI Studio\n",
+    "* Go to Azure AI Studio (ai.azure.com)\n",
+    "* Select Meta-Llama-3.1-405B-Instruct model from Model catalog.\n",
+    "* Deploy with \"Pay-as-you-go\"\n",
+    "* Once deployed successfully, you should be assigned for an API endpoint and a security key for inference.\n",
+    "\n",
+    "Update the following cell with the information of the deployment you just created."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Llama-3-405B Teacher model endpoint name\n",
+    "# The serverless model name is the name found in ML Studio > Endpoints > Serverless endpoints > Model column\n",
+    "TEACHER_MODEL_NAME = \"Meta-Llama-3.1-405B-Instruct\"\n",
+    "\n",
+    "# The serverless model endpoint name is the name found in ML Studio > Endpoints > Serverless endpoints > Name column\n",
+    "# The endpoint URL will be resolved from this name by the MLFlow component\n",
+    "TEACHER_MODEL_ENDPOINT_NAME = \"Meta-Llama-3-1-405B-Instruct-vum\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Pick a student model\n",
+    "\n",
+    "We will use **Meta-Llama-3.1-8B-Instruct** as student model. We only support chat completion models that are available for PayGo finetuning in Azure AI Studio."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "STUDENT_MODEL_NAME = \"Meta-Llama-3.1-8B-Instruct\"\n",
+    "STUDENT_MODEL_VERSION = 1\n",
+    "\n",
+    "# retrieve student model from model registry\n",
+    "mlclient_azureml_meta = MLClient(credential, registry_name=\"azureml-meta\")\n",
+    "student_model = mlclient_azureml_meta.models.get(\n",
+    "    STUDENT_MODEL_NAME, version=STUDENT_MODEL_VERSION\n",
+    ")\n",
+    "\n",
+    "print(\n",
+    "    \"\\n\\nUsing model name: {0}, version: {1}, id: {2} for fine tuning\".format(\n",
+    "        student_model.name, student_model.version, student_model.id\n",
+    "    )\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Download the dataset from HuggingFace repo\n",
+    "\n",
+    "For our example, we download and use the Common Sense Q&A dataset (https://huggingface.co/datasets/tau/commonsense_qa/) from HuggingFace."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from datasets import load_dataset\n",
```

Comment:
Could we please move these to utils files?

Reply:
@skasturi - Its updated